texar/torch/modules/pretrained/pretrained_base.py

```diff
@@ -29,30 +29,49 @@
 
 __all__ = [
     "default_download_dir",
+    "set_default_download_dir",
```

Comment:
Should we move these two functions to `texar`-level utils?  I think `Tokenizer` modules will need these functions to load the pre-trained vocabulary.

Reply:
Would we have tokenizers under a separate package, or will they also be under `pretrained`? I think we can keep these functions here until the tokenizers are implemented.

Reply:
I planed to make them under `modules/tokenizers`. Sure, we can keep them as they are and move them if necessary in the future.

Reply:
Since the users will access these functions via `tx.modules.default_...`, moving them to `modules/tokenizers` won't affect them. It might be a different story if tokenizers are put under `data/`.